### ISSUE TITLE
Fraction bars disappear when printing webpages into PDFs.

### ISSUE THREAD

**Issue by riteme** [Khan/KaTeX#1593]
Hello, I'm using *KaTeX 0.10.0-beta* and a fresh installation of Google Chrome Beta 69.0.3497.23 on Ubuntu 18.04 (to make sure that all preferences are defaults). I'm considering printing webpages with math formulas into PDFs. But I came across some issues with fractions.

For example:

```html
<!DOCTYPE html>
<html>
<head>
  <link href="katex/katex.min.css" type="text/css" rel="stylesheet">
  <script src="katex/katex.min.js" type="text/javascript"></script>
  <script src="katex/contrib/auto-render.min.js" type="text/javascript"></script>
  <script>
    document.addEventListener("DOMContentLoaded", function() {
      renderMathInElement(document.body, options = {delimiters: [{ left: "$$", right: "$$", display: true }, { left: "$", right: "$", display: false }]});
    });
  </script>
</head>
<body>
<p>inline math: ${2 \over n}$.</p>
<p>display math:
$$
f^{(n)}(x) = \begin{cases}
x + nb &amp; (a = 1) \\
a^n\ x + {a^n - 1 \over a - 1} \cdot b &amp; (a &gt; 1)
\end{cases}
$$
</p>
</body>
</html>
```

Print it via Chrome and will get:

![](https://gitee.com/riteme/blogimg/raw/master/other/katex-print-1.png)

Press "F12" to add the following CSS code:

```css
.katex {
    font-size: 1em;  /* or even smaller */
}
```

Reprint and get:

![](https://gitee.com/riteme/blogimg/raw/master/other/katex-print-2.png)

I have also tried <https://khan.github.io/KaTeX/> and set `font-size` to `0.3em` and came across the same issue.  I wonder why printed PDFs would be different from original webpages. Is that a bug in Chrome or KaTeX?

**Comment by riteme** [comment]
Fraction bars begin to show after the CSS added. However it seems that horizontal lines are too wide:

![](https://gitee.com/riteme/blogimg/raw/master/other/katex-print-4.png)

Is that a bug with Chrome? Since the underline of "display math" has the same width.

**Comment by ronkok** [comment]
> Is that a bug with Chrome?

We've had no reports that I recall of this type from Firefox or Edge. Safari has similar problems.

>  it seems that horizontal lines are too wide.

Try `0.5px`. Let me know how this turns out. I'll write a PR that adds the best results to KaTeX.

**Comment by mbourne** [comment]
To add on here, for print, I normally use a fixed `font-size:10pt`.

Fractions work fine in PDF (generated from Chrome) at that setting:

![image](https://user-images.githubusercontent.com/7541940/43998243-bb986e28-9e23-11e8-94bc-cd87b0f278ff.png)

**Comment by ronkok** [comment]
@mbourne Do you think the ` min-height: 1px;` makes lines that are too thick in print mode?

**Comment by riteme** [comment]
@ronkok I finally find out that with a larger `font-size` the relative thicknesses become reasonable. No matter what unit is used. Either `3em` or `39px` looks fine on my screen.

**Comment by mbourne** [comment]
@ronkok Sorry about the delay. I found that my earlier optimism was misplaced. (I never actually print things - I was basing my earlier comment on the screen shot of the PDF generated by Chrome.)
When doing an actual print, even with 10pt fixed, I saw the disappearing lines that @riteme reported.
Fixing the border to `min-height:1px` did the trick, at least for the pages I tried. They were not too thick - looked good to me.
